package.json

```diff
@@ -107,12 +119,13 @@
     "pofile": "1.0.10",
     "postcss-flexbugs-fixes": "4.1.0",
     "postcss-loader": "3.0.0",
+    "prepend-http": "2.0.0",
     "prettier": "1.14.3",
     "promise-file-reader": "1.0.2",
     "query-string": "6.1.0",
     "raven": "2.6.4",
     "raven-js": "3.27.0",
-    "razzle": "2.4.0",
+    "razzle": "3.0.0-alpha.0",
```

Comment:
@nileshgulia1 we might want to wait for a final version of Razzle here. I guess this upgrade is necessary for Babel 7? Or does Razzle 2.4 work with Babel 7 as well?

Reply:
@tisto We are having babel 7 presets with razzle v3 . But since we're using razzle's presets for babel, for v7 we have to use razzle v3, otherwise we may opt out using razzle's babel configs.